app/src/main/java/fr/free/nrw/commons/profile/achievements/AchievementsFragment.java

```diff
@@ -456,44 +465,51 @@ private void hideProgressBar(Achievements achievements) {
 
     @OnClick(R.id.images_upload_info)
     public void showUploadInfo(){
-        launchAlert(getResources().getString(R.string.images_uploaded)
-                ,getResources().getString(R.string.images_uploaded_explanation));
+        launchAlertWithHelpLink(getResources().getString(R.string.images_uploaded)
+                ,getResources().getString(R.string.images_uploaded_explanation)
+            ,IMAGES_UPLOADED_URL);
     }
 
     @OnClick(R.id.images_reverted_info)
     public void showRevertedInfo(){
-        launchAlert(getResources().getString(R.string.image_reverts)
-                ,getResources().getString(R.string.images_reverted_explanation));
+        launchAlertWithHelpLink(getResources().getString(R.string.image_reverts)
+                ,getResources().getString(R.string.images_reverted_explanation)
+            ,IMAGES_REVERT_URL);
     }
 
     @OnClick(R.id.images_used_by_wiki_info)
     public void showUsedByWikiInfo(){
-        launchAlert(getResources().getString(R.string.images_used_by_wiki)
-                ,getResources().getString(R.string.images_used_explanation));
+        launchAlertWithHelpLink(getResources().getString(R.string.images_used_by_wiki)
+                ,getResources().getString(R.string.images_used_explanation)
+            ,IMAGES_USED_URL);
```

Comment:
Spaces after comma.
This makes code more readable.

Reply:
@4D17Y4 is right that space is needed after every comma.
Actually the best here would be to put the commas at the end of each line.
Also, indentation should be the same for arguments of the same level.
Not a hard convention but when there are arguments over several lines and most arguments are rather long, for readibility I suggest using one line per argument.
In other words, I recommend changing to the following, thanks!
<img width="457" alt="Screen Shot 2021-04-03 at 11 24 27" src="https://user-images.githubusercontent.com/99590/113465471-2e0fcc00-946f-11eb-9457-23584e77f169.png">